info.json

```diff
@@ -0,0 +1,8 @@
+{
+  "Id": "MOD_NAME",
+  "Version": "0.1.0",
+  "DisplayName": "MOD_NAME",
+  "Author": "Your Name",
```

Comment:
I can't figure out how to suggest edits like you did for me, but I'd suggest using `MOD_AUTHOR` here instead.

Reply:
You can suggest edits by doing

```
```suggestion
New code here
```